### PR TITLE
`azurerm_spring_cloud_customized_accelerator` - support for the `accelerator_type` and `sub_path`

### PR DESCRIPTION
**Pull request by ms-henglu**
image

### CODE REVIEW

File: website/docs/r/spring_cloud_customized_accelerator.html.markdown [hunk 1]
```diff
@@ -91,6 +93,8 @@ A `git_repository` block supports the following:
 
 * `ssh_auth` - (Optional) A `ssh_auth` block as defined below. Conflicts with `git_repository.0.basic_auth`. Changing this forces a new Spring Cloud Customized Accelerator to be created.
 
+* `sub_path` - (Optional) Specifies the folder path inside the git repository to consider as the root of the accelerator or fragment.
```
Comment (magodo) [inline website/docs/r/spring_cloud_customized_accelerator.html.markdown:96]: The `sub_path` can probably be shortend to just `path`. The description can then be reworded to:

```suggestion
* `path` - (Optional) Specifies the path under the git repository to be treated as the root directory of the accelerator or the fragment (depending on `accelerator_type`).
```

**Review by magodo — CHANGES_REQUESTED** [review]
Thanks for this PR!
I have left some additional mostly minor comments that once addressed this should be good to merge 👍

File: website/docs/r/spring_cloud_customized_accelerator.html.markdown (hunk 1 above)
Comment (ms-henglu) [reply website/docs/r/spring_cloud_customized_accelerator.html.markdown]: Thanks! I've updated it.

**Review by magodo — APPROVED** [review]

**Review by stephybun — APPROVED** [review]
Thanks @ms-henglu and @magodo LGTM 👍

**Pull request merged**